website/src/pages/tutorial.md

```diff
@@ -0,0 +1,523 @@
+## Introduction
+
+In this hands-on tutorial, we will take a look at how we can use Yew to build web applications.
+**Yew** is a modern [Rust](https://www.rust-lang.org/) framework for building front-end web apps using [WebAssembly](https://webassembly.org/).
+Yew encourages a reusable, maintainable, and well-structured architecture by leveraging Rust's powerful type system.
+A large ecosystem of community-created libraries (or crates?) provide components that work out-of-the-box and
+commonly-used patterns such as state management.
+
+Using WebAssembly also allows us to take advantage of numerous crates available on [crates.io](https://crates.io)
+
+### What we are going to build
+
+Rustconf is an intergalactic gathering of the Rust community that happens annually.
+Rustconf 2020 had a plethora of talks that provided a good amount of information.
+In this hands on tutorial, we will be building a web application to help fellow Rustaceans
+have overview of the talks, and watch them all from one page.
+
+## Setting up
+
+### Prerequisites
+
+To get started, let's make sure we have an up-to-date development environment.
+We will need the following tools:
+- [Rust](https://www.rust-lang.org/).
+- [`trunk`](https://trunkrs.dev/)
+
+This tutorial also assumes you're already familiar with Rust.
```

Comment:
- [Rust](https://www.rust-lang.org/)
- [Trunk](https://trunkrs.dev/)
- `wasm32-unknown-unknown`, the WASM compiler and build target for Rust

Ensure the latest version of Rust is installed by running `rustup update` or by 
[installing rust](https://www.rust-lang.org/tools/install) if you haven't already done so already.

After installing Rust, you can use Cargo to install `trunk` by running:

```bash
cargo install trunk
```

We will also need to add the WASM build target by running:

```bash
rustup target add wasm32-unknown-unknown
```
```
- [Rust](https://www.rust-lang.org/)
- [Trunk](https://trunkrs.dev/)
- `wasm32-unknown-unknown`, the WASM compiler and build target for Rust

Ensure the latest version of Rust is installed by running `rustup update` or by 
[installing rust](https://www.rust-lang.org/tools/install) if you haven't already done so already.

After installing Rust, you can use Cargo to install `trunk` by running:

\`\`\`bash
cargo install trunk
\`\`\`

We will also need to add the WASM build target by running:

\`\`\`bash
rustup target add wasm32-unknown-unknown
\`\`\`
```